Add Sonata blog post. (#4)
CHERIoT-Platform/CHERIoT-Platform.github.io: _posts/2024-06-10-sonata-quick-start.markdown

If you open the RTOS repository in a GitHub Codespace, you can start working in this environment immediately.

To be able to load firmware onto a Sonata board from the container, you need to expose the `SONATA` filesystem into the container.
This requires you to add a section like this to the [`devcontainer.json`](https://github.com/microsoft/cheriot-rtos/blob/main/.devcontainer/devcontainer.json) file:

```json
  "mounts": [
    "source=/Volumes/SONATA,target=/mnt/SONATA,type=bind"
  ]
```

Unfortunately, we can't do that automatically because the source needs to be the mount point on your host.
On Windows, for example, this may be something like `E:`, on Linux it's probably somewhere under `/run/media` or `/mnt`.

VS Code will automatically install and configure extensions to make working with CHERIoT easy.
For example, you'll notice that autocomplete and code cross-referencing work out of the box, including understanding the CHERIoT-specific C/C++ attributes.

You can now build any of the examples, exercises, or the test suite by running the following commands in the project's directory:

```sh
$ xmake config --sdk=/cheriot-tools --board=sonata
$ xmake
$ xmake run
```

> If you didn't add the `SONATA` mount point, don't worry.
> You can't do that in the GitHub Codespace, but you can still use the code space for development.
> The `xmake run` output will look something like this:

Reviewer: ```suggestion
If you didn't add the `SONATA` mount point or you are in a GitHub Codespace where you can't, don't worry.
The `xmake run` output will look something like this:
```
